src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(person => person.sex === 'm').filter(
+      person => Math.ceil(person.died / 100) === century
+    )
+    : people.filter(person => person.sex === 'm');
```

Comment:
You can create a helper function here, cuz U use it several times - here and to find women)